operations/patch_util.go

```diff
@@ -586,7 +586,7 @@ func diffToMbox(diffData *localDiff, subject string) (string, error) {
 }
 
 func addMetadataToDiff(diffData *localDiff, metadata GitMetadata) (string, error) {
-	mboxTemplate, err := template.New("mbox").Parse(`From 72899681697bc4c45b1dae2c97c62e2e7e5d597b Mon Sep 17 00:00:00 2001
+	mboxTemplate := template.Must(template.New("mbox").Parse(`From 72899681697bc4c45b1dae2c97c62e2e7e5d597b Mon Sep 17 00:00:00 2001
```

Comment:
This panics if it doesn't work, is that preferable to an error?

Reply:
I would argue that side: Either I made a programming error and it will always panic, or I didn't and it never will.
That said, I don't have string feelings. It looks like we use it this way about 1/3 of the time and we handle the error the other 2/3.

Reply:
Yeah I don't have strong feelings, sounds good 👍